File: PSet1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

// function prototype
int get_size();
void print_pyramid(int size);

int main(void)
{
    // get the size of pyramid
    int size;
    size = get_size();

    // print the pyramid
    print_pyramid(size);
}

// get user input
int get_size()
{
    int size;
    do
    {
        size = get_int("Height: ");
    }
    while (size < 1 || size > 8);

    return size;
}

// print the pyramid
void print_pyramid(int size)
{
    for (int i = 0; i < size; i ++)
    {
        for (int j = 0; j < size; j ++)
        {
            if (j < size - i - 1)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        printf("\n");
    }
}

File: Pset6/sentimental-mario-less/mario.py
# import cs50 module
from cs50 import get_int


def get_height():
    # get user's input of height
    while True:
        height = get_int("Height: ")
        if 1 <= height <= 8:
            return height


def main():
    # main function
    height = get_height()
    for i in range(height):
        print(" " * (height - i - 1), end="")
        print("#" * (i + 1))


if __name__ == '__main__':
    main()

